Exercícios/Para casa/entregas/exemplo-nome-sobrenome/Maneger.js

```diff
@@ -0,0 +1,24 @@
+const { Person } = require("./Person");
+
+class Manager extends Person {
+  #salary;
+  clients;
+
+  constructor(name, cpf, salary) {
+    super(name, cpf), (this.#salary = salary), (this.clients = []);
```

Comment:
Oi, Ellie. 😄Gostei como você desenvolveu o desafio, bem estruturado e limpo. Muito bom a refatoração! 🚀`constructor(name, cpf, salary) {
    super(name, cpf), (this.#salary = salary), (this.clients = []); ` essa parte é o mesmo que  `constructor(name, cpf, salary) {
    super(name, cpf), this.#salary = salary, this.clients = []}; `?